string/template filler/d57766a1-8ced-4ed9-ba56-f6b3988d7d67.json

```diff
@@ -4,17 +4,20 @@
     "inputs": {
       "25aa567c-3316-4206-b42e-4cfe6a67b0c7": {
         "type": "string",
-        "name": "template"
+        "name": "template",
+        "description": "The template to be filled"
```

Comment:
I think an example for the port values would be highly suggested here as it is not trivial how this node should be used.
Also the general description lacks examples.